File: Coursera - Build Your First Android App/Block03 - Linear and Relative Layouts/app/src/main/java/com/example/block3/MainActivity.java
package com.example.block3;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState); // Menjalankan fungsi onCreate -> membuat tampilan

        setContentView(R.layout.activity_main); // Men-set isi tampilan dengan layout dari file xml (activity_main)
    }
}

File: z01_miscellaneous/firstApp - animasi 2/app/src/main/java/com/example/firstapp/MainActivity.kt
package com.example.firstapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var garutIsShowing = true

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun garutOnClick(view: View) {
        Log.i("info", "garut telah di klik")

        var garut = garutImageView
        var nanas = nanasImageView

        if(garutIsShowing) {
            garut.animate().rotation(360f).scaleX(0.5f).scaleY(0.5f).alpha(1f).setDuration(2000)
            garutIsShowing = false
        } else {
            garut.animate().rotation(-360f).scaleX(1f).scaleY(1f).alpha(0f).setDuration(2000)
            garutIsShowing = true
        }
    }
}

